kubernetes-javascript/index.js

```diff
@@ -12,3 +12,4 @@ const deployment = new k8s.apps.v1.Deployment("nginx", {
         }
     }
 });
+export const name = deployment.metadata.apply(m => m.name);
```

Comment:
This won’t work in JavaScript. We use `export.name = ` instead.

Reply:
Ah right, whoops.

Reply:
Well, technically, it _does_ work in modern JavaScript. But to be consistent, I'll change to `exports.name`.

Reply:
Unfortunately, Node.js, which is the host runtime for Pulumi, does not support this in `.js` files and has no plans to. 

https://nodejs.org/api/esm.html